File: FertiWeb/src/main/java/dominio/TipoFuente.java
package dominio;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class TipoFuente {

	private Long codigo;
	private String nombre;
	private double aporte;
	private Long unidad;
	private boolean estado;
	private Long codigoElemento;

}

File: FertiFront/src/app/clases_dominio/tipo-fuente.ts
import { Unidad } from './unidad';
import { Elemento } from './elemento';

export class TipoFuente {
    public codigo: number;
    public nombre: string;
    public aporte: number;
    public estado: boolean;
    public unidad: Unidad;
    public elemento: Elemento;
}
